scaling_performance/optimizing_ephemeral_storage.adoc

```diff
@@ -0,0 +1,112 @@
+[[scaling-performance-optimizing-ephemeral-storage]]
+= Optimizing Ephemeral Storage
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+Ephemeral (transient) storage is used by pods for their internal
+operation, such as for temporary files.  The lifetime of this
+ephemeral storage does not extend beyond the life of the individual
```

Comment:
Except in the case of empty dir and logs.

Reply:
+1